src/core/public/chrome/ui/header/branding/opensearch_dashboards_custom_logo.tsx

```diff
@@ -59,18 +61,50 @@ export interface CustomLogoType {
  *          the default opensearch logo will be rendered.
  */
 export const CustomLogo = ({ ...branding }: CustomLogoType) => {
-  const headerLogoUrl = !branding.logo.defaultUrl
-    ? branding.mark.defaultUrl
-    : branding.logo.defaultUrl;
-  return !branding.logo.defaultUrl && !branding.mark.defaultUrl ? (
+  const darkmode = branding.darkMode;
+  const logoDefault = branding.logo.defaultUrl;
+  const logoDarkMode = branding.logo.darkModeUrl;
+  const markDefault = branding.mark.defaultUrl;
+  const markDarkMode = branding.mark.darkModeUrl;
+  const applicationTitle = branding.applicationTitle;
+
+  const customHeaderLogoDefaultMode = () => {
```

Comment:
think it would be helpful to give a function header comment to show expected functionality.

Reply:
Added. Thx!